src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -1,153 +1,159 @@
-/*

```

Comment:
As written, both files on HDD have CRLF.
Will try it with to converting forth and back...